aerosonicdb/models/mfcc/cnn.py

```diff
@@ -1,55 +1,60 @@
 import os
+
+import absl.logging
+import matplotlib.pyplot as plt
 import numpy as np
-import tensorflow.keras as keras
 import tensorflow as tf
-import matplotlib.pyplot as plt
 from scikeras.wrappers import KerasClassifier
+from sklearn.metrics import PrecisionRecallDisplay, average_precision_score
 from sklearn.model_selection import cross_validate
-from sklearn.metrics import average_precision_score, PrecisionRecallDisplay
-from aerosonicdb.utils import get_project_root
-from aerosonicdb.utils import fetch_k_fold_cv_indicies, train_val_split
-from aerosonicdb.utils import load_train_data, load_test_data, load_env_test_data
-import absl.logging
+from tensorflow import keras
+from tensorflow.keras import Sequential, layers
+
+from aerosonicdb.utils import (
+    fetch_k_fold_cv_indicies,
+    get_project_root,
+    load_env_test_data,
+    load_test_data,
+    load_train_data,
+    train_val_split,
+)
+
 absl.logging.set_verbosity(absl.logging.ERROR)
 tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.FATAL)
 
 
 ROOT_PATH = get_project_root()
-FEAT_PATH = os.path.join(ROOT_PATH, 'data/processed')
-TRAIN_PATH = os.path.join(FEAT_PATH, '13_mfcc_5_train.json')
-TEST_PATH = os.path.join(FEAT_PATH, '13_mfcc_5_test.json')
-ENV_FEAT_BASE = '_ENV_13_mfcc_5.json'
-OUTPUT_PATH = os.path.join(ROOT_PATH, 'models')
+FEAT_PATH = os.path.join(ROOT_PATH, "data/processed")
+TRAIN_PATH = os.path.join(FEAT_PATH, "13_mfcc_5_train.json")
+TEST_PATH = os.path.join(FEAT_PATH, "13_mfcc_5_test.json")
+ENV_FEAT_BASE = "_ENV_13_mfcc_5.json"
+OUTPUT_PATH = os.path.join(ROOT_PATH, "models")
 
 
 def build_model(input_shape):
-
-    model = keras.Sequential()
-
-    # 1st conv layer
-    model.add(keras.layers.Conv2D(32, (3, 3), activation='relu', input_shape=input_shape))
-    model.add(keras.layers.MaxPooling2D((3, 3), strides=(2, 2), padding='same'))
-    model.add(keras.layers.Dropout(0.4))
-
-    # 2nd conv layer
-    model.add(keras.layers.Conv2D(32, (3, 3), activation='relu'))
-    model.add(keras.layers.MaxPooling2D((3, 3), strides=(2, 2), padding='same'))
-    model.add(keras.layers.Dropout(0.4))
-
-    # 3rd conv layer
-    model.add(keras.layers.Conv2D(32, (2, 2), activation='relu'))
-    model.add(keras.layers.MaxPooling2D((2, 2), strides=(2, 2), padding='same'))
-    model.add(keras.layers.Dropout(0.4))
-
-    # flatten and feed to dense layer
-    model.add(keras.layers.Flatten())
-    model.add(keras.layers.Dense(32, activation='relu'))
-    model.add(keras.layers.Dropout(0.4))
-
-    # output layer
-    model.add(keras.layers.Dense(1, activation='sigmoid'))
-
-    return model
+    """Define CNN model architecture."""

```

Comment:
just a more compact definition that makes it easy to appreciate model architecture at a glance and change things potentially in the future (hyper parameters etc)..

I saw you already use this format for the `mlp` model.